src/main.cu

```diff
@@ -128,7 +126,7 @@ int main()
     //create_world
 
     // Load object
-    const char *file_path = "models/cube.obj";
+    const char *file_path = "models/cubes.obj";

```

Comment:
Let's revert that change, not relevant in this PR and might be problematic for my local setup 😬

Reply:
yikes

Reply:
hmmm with 1 cube is hard to see the effect of light